src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,110 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+    private User validUser;
+
+    @BeforeEach
+    public void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        validUser = new User();
+        validUser.setLogin("validLogin");
+        validUser.setPassword("validPassword");
+        validUser.setAge(25);
+        Storage.people.clear(); // Clear storage before each test
```

Comment:
Ensure that `Storage.people.clear()` effectively clears the storage to prevent test interference. This is crucial for maintaining test isolation.